src/complex/Utilities/SIMPLConversion.hpp

```diff
@@ -0,0 +1,2466 @@
+    value.inputFilePath = json[k_InputFileKey].get<std::string>();
+    value.startSlice = json[k_ZStartIndexKey].get<int32>();
+    value.endSlice = json[k_ZEndIndexKey].get<int32>();
+    value.eulerRepresentation = json[k_RefFrameZDirKey].get<int32>();
+    value.useRecommendedTransform = static_cast<bool>(json[k_UseTransformationsKey].get<int32>());
+    value.hdf5DataPaths = json[k_SelectedArrayNamesKey].get<std::vector<std::string>>();
+
+    return {std::move(value)};
+  }
+};
+#endif
+
+struct DataContainerReaderFilterParameterConverter
+{
+  using ParameterType = Dream3dImportParameter;
+  using ValueType = ParameterType::ValueType;
+
+  static constexpr StringLiteral k_InputFileKey = "InputFile";
+  static constexpr StringLiteral k_InputFileDataContainerArrayProxyKey = "InputFileDataContainerArrayProxy";
+  static constexpr StringLiteral k_DataContainerProxyKey = "Data Containers";
+  static constexpr StringLiteral k_AttributeMatrixProxyKey = "Attribute Matricies";
+  static constexpr StringLiteral k_DataArrayProxyKey = "Data Arrays";
+  static constexpr StringLiteral k_DataObjectNameKey = "Name";
+
+  static Result<ValueType> convert(const nlohmann::json& json)
+  {
+    if(!json[k_InputFileKey].is_string())
+    {
+      return MakeErrorResult<ValueType>(-2, fmt::format("H5EbsdReaderParameterConverter json '{}' is not a string", json[k_InputFileKey].dump()));
+    }
+    if(!json[k_InputFileDataContainerArrayProxyKey].is_object())
+    {
+      return MakeErrorResult<ValueType>(-1, fmt::format("H5EbsdReaderParameterConverter json '{}' is not an object", json[k_InputFileDataContainerArrayProxyKey].dump()));
+    }
+
+    std::string inputFilePath = json[k_InputFileKey].get<std::string>();
+    std::vector<complex::DataPath> dataPaths;
+
+    auto dcaProxyJson = json[k_InputFileDataContainerArrayProxyKey];
+    if(!dcaProxyJson[k_DataContainerProxyKey].is_array())
+    {
+      return MakeErrorResult<ValueType>(-3, fmt::format("H5EbsdReaderParameterConverter json '{}' is not an array", dcaProxyJson[k_DataContainerProxyKey].dump()));
+    }
+
+    for(const auto& dcIter : dcaProxyJson[k_DataContainerProxyKey])
+    {
+      std::string dcName = dcIter[k_DataObjectNameKey].get<std::string>();
+      DataPath dcPath({dcName});
+      dataPaths.push_back(dcPath);
+
+      for(const auto& amIter : dcIter[k_AttributeMatrixProxyKey])
+      {
+        std::string amName = amIter[k_DataObjectNameKey].get<std::string>();
+        DataPath amPath = dcPath.createChildPath(amName);
+        dataPaths.push_back(amPath);
+
+        for(const auto& daIter : amIter[k_DataArrayProxyKey])
+        {
+          std::string daName = daIter[k_DataObjectNameKey].get<std::string>();
+          DataPath daPath = amPath.createChildPath(daName);
+          dataPaths.push_back(daPath);
+        }
+      }
+    }
+
+    ParameterType::ValueType value;
+    value.FilePath = std::filesystem::path(inputFilePath);
+    value.DataPaths = dataPaths;
```

Comment:
```suggestion
    value.DataPaths = std::move(dataPaths);
```
This should prevent double allocation of memory for an object of objects